### ISSUE TITLE
Extremely slow

### ISSUE THREAD

**Issue by Mikanoshi** [Stericson/RootTools#37]
HTC One Sense 6, 5.0.2
Updated from 3.4 to 4.2 and now all commands are very slow taking ~2 seconds.
Code example:
https://gist.github.com/Mikanoshi/a2f4a6de794f403f2297

Debug log:
```
03-06 09:12:56.519: D/RootShell v1.3(24730): CommandHandler created
03-06 09:12:56.519: D/RootShell v1.3(24730): Using Existing Shell!
03-06 09:12:56.529: D/RootShell v1.3(24730): Executing: cat /sys/class/leds/button-backlight/currents with context: NORMAL
03-06 09:12:56.529: D/RootShell v1.3(24730): Waiting for output to be processed. 0 Of 1
03-06 09:12:56.609: D/Command(24730): ID: 0, 20
03-06 09:12:56.609: D/RootShell v1.3(24730): CommandHandler created
03-06 09:12:56.609: D/RootShell v1.3(24730): Using Existing Shell!
03-06 09:12:56.619: D/RootShell v1.3(24730): Executing: stat -c '%u' /sys/class/leds/button-backlight/currents with context: NORMAL
03-06 09:12:58.531: D/RootShell v1.3(24730): Read all output
03-06 09:12:58.531: D/RootShell v1.3(24730): Command 0 finished.
03-06 09:12:58.531: D/RootShell v1.3(24730): Waiting for output to be processed. 0 Of 1
03-06 09:12:58.531: D/Command(24730): ID: 0, 10253
03-06 09:12:58.541: D/RootShell v1.3(24730): Using Existing Shell!
03-06 09:12:58.541: D/RootShell v1.3(24730): CommandHandler created
03-06 09:12:58.541: D/RootShell v1.3(24730): Executing: chmod 644 /sys/class/leds/button-backlight/currents
03-06 09:12:58.541: D/RootShell v1.3(24730): echo 0 > /sys/class/leds/button-backlight/currents
03-06 09:12:58.541: D/RootShell v1.3(24730): chmod 444 /sys/class/leds/button-backlight/currents with context: NORMAL
03-06 09:13:00.544: D/RootShell v1.3(24730): Read all output
03-06 09:13:00.544: D/RootShell v1.3(24730): Command 0 finished.
03-06 09:13:00.544: D/RootShell v1.3(24730): Read all output
03-06 09:13:00.544: D/RootShell v1.3(24730): Command 0 finished.
```
Is something wrong with executing commands like that? This is the same code with RootTools 3.5:
```
03-06 17:32:31.466: D/RootTools v3.5(20683): CommandHandler created
03-06 17:32:31.466: D/RootTools v3.5(20683): Using Existing Shell!
03-06 17:32:31.486: D/RootTools v3.5(20683): Executing: cat /sys/class/leds/button-backlight/currents
03-06 17:32:31.486: D/RootTools v3.5(20683): Command 0 finished.
03-06 17:32:31.556: D/RootTools v3.5(20683): CommandHandler created
03-06 17:32:31.556: D/RootTools v3.5(20683): Using Existing Shell!
03-06 17:32:31.566: D/RootTools v3.5(20683): Executing: stat -c '%u' /sys/class/leds/button-backlight/currents
03-06 17:32:31.566: D/RootTools v3.5(20683): Using Existing Shell!
03-06 17:32:31.566: D/RootTools v3.5(20683): CommandHandler created
03-06 17:32:31.566: D/RootTools v3.5(20683): Command 0 finished.
03-06 17:32:31.576: D/RootTools v3.5(20683): Executing: chmod 644 /sys/class/leds/button-backlight/currents
03-06 17:32:31.576: D/RootTools v3.5(20683): echo 0 > /sys/class/leds/button-backlight/currents
03-06 17:32:31.576: D/RootTools v3.5(20683): chmod 444 /sys/class/leds/button-backlight/currents
03-06 17:32:31.596: D/RootTools v3.5(20683): Command 0 finished.
```

**Comment by kostaspl** [comment]
@Mikanoshi 
I think you should reduce the waiting time in Shell.java, line 733.
https://github.com/Stericson/RootShell/blob/master/src/com/stericson/RootShell/execution/Shell.java#L733

**Comment by Mikanoshi** [comment]
@kostaspl The answer is in that gist. It's either disabling this delay (still no idea why it is required, 3.5 was working perfectly without any delays) or disable handler (that's what I tried).
With handler disabled commands work absolutely unreliably, commandOutput() is called only for the first command when executing them one after another. Also isRootAvailable() and isBusyboxAvailable() are executed like 10 seconds after the actual call sometimes...

TL;DR 4.2 is broken for me, I've returned to 3.5

**Comment by Mikanoshi** [comment]
Looks like it's working fine after disabling handler.

**Comment by chloeontou** [comment]
fuckin hackers credit cards etc
